test/compiler/codegen.jl

```diff
@@ -707,7 +707,19 @@ mktempdir() do pfx
     errfile = joinpath(pfx, "stderr.txt")
     @test libs_emptied > 0
     @test_throws ProcessFailedException run(pipeline(`$pfx/bin/$(Base.julia_exename()) -e 'print("This should fail!\n")'`; stderr=errfile))
-    @test contains(readline(errfile), "ERROR: Unable to load dependent library")
+    utf16 = bytes -> begin
+        n = Int(length(bytes)//2)
+        pairs = [bytes[(2i-1):(2i)] for i in 1:n]
+        f = pair -> reinterpret(UInt16, pair)
+        units = only.(f.(pairs))
+        return transcode(String, units)
+    end
+    @static if Sys.iswindows()
+        errmsg = utf16(read(errfile))
+    else
+        errmsg = readline(errfile)
+    end
```

Comment:
Alternatively:
```julia
errmsg = replace(readline(errfile), "\0" => "")
```
? :grin:  It's less accurate because it blindly drops all `"\0"`, wherever they are, but it's also much simpler.

Reply:
@apaz-cli FYI, I believe that's the right way (and much faster):
``` 
n = length(bytes) ÷ 2
```

See also div and \div TAB to type it. With rationals it's not also exactly the same, can throw, with odd number of bytes, which shouldn't happen... maybe you used it on purpose for that?